build.sbt

```diff
@@ -738,6 +740,15 @@ lazy val dlcOracleTest = project
   )
   .dependsOn(coreJVM % testAndCompile, dlcOracle, testkit)
 
+lazy val oracleExplorerClient = project
+  .in(file("oracle-explorer-client"))
+  .settings(CommonSettings.settings: _*)
+  .settings(
+    name := "bitcoin-s-oracle-explorer-client",
+    libraryDependencies ++= Deps.oracleExplorerClient
+  )
+  .dependsOn(coreJVM, appCommons, testkit % "test->test")
```

Comment:
this should be split into `oracle-explorer-client` & `oracle-explorer-client-test` so any project that wants to use this doesn't need testkit

Reply:
So this isn't quite right. 

You can inspect classpath's by doing `show fullClasspath` and `show test:fullClasspath` to see the difference between the `src` classpath and the `test` class path. The reason this doesn't show up is the `test->test` qualifier added to the end of `testkit`

You will see that the testkit deps show up on `test:fullClasspath`, but if you just do `fullClasspath` you won't see them. 

The reason we _normally_ don't want to included tests and source files in the same module is because we _want_ to use the sources in the testkit project to create fixtures. 

For example, we create bitcoind fixtures inside of testkit to _test_ our implementations of the exact same `BitcoindRpcClient`. 

https://github.com/bitcoin-s/bitcoin-s/blob/478ab16e28c65a4796f92433a6fef12cd35e351a/testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindFixtures.scala#L43

To do this, we need the testkit project to act as a middle man between the two. 

We could have just written all the test harness code inside of `bitcoind-rpc/src/test` but then it isn't easily re-usable (and publishable as a standalone dependency -- `testkit`) for other modules.
 
If in the future we want use the explorer http client inside our test fixtures to _test_ the `SbExplorerClient` implementation, we would need to do this.